r1cs/gadgets/std/src/bits/boolean.rs

```diff
@@ -699,6 +705,230 @@ impl Boolean {
 
         Ok(())
     }
+
+    /// Computes (a and b) xor ((not a) and c)
+    pub fn sha256_ch<'a, ConstraintF, CS>(
```

Comment:
Can we move this sha256 related functions in its corresponding file ? This Boolean struct is supposed to be general purpose

Reply:
In Bellman they are in boolean.rs too.

Reply:
We are more general purpose than Bellman

Reply:
Ok, I will move and rename the two implementation for Boolean and UInt32

Reply:
Moved in sha256.rs